### PR TITLE
Fix `bouncingBar` spinner

### PR DESCRIPTION
**Pull request by GeorgeTG**

### CODE REVIEW

File: spinners.json [hunk 1]
```diff
@@ -659,14 +659,21 @@
 	"bouncingBar": {
 		"interval": 80,
 		"frames": [
-			"[    ]",
+            		"[    ]",
```
Comment (sindresorhus) [inline spinners.json]: I don't think there should be two empty frames.

**Comment by sindresorhus** [comment]
You need to fix the indentation. See the diff.

File: spinners.json (hunk 1 above)
Comment (GeorgeTG) [inline spinners.json]: It has only one, in the middle

Comment (GeorgeTG) [inline spinners.json]: I think it looks fine like that

**Comment by SamVerschueren** [comment]
Indentation is still incorrect. You are using spaces instead of tabs. Install the [editorconfig](http://editorconfig.org/) plugin for your editor and it will automatically pick up the correct settings.

**Pull request merged**